draft-ietf-mls-architecture.md

```diff
@@ -291,35 +291,36 @@ While informally, a group can be considered to be a set of users
 possibly using multiple endpoint devices to interact with the
 Service Provider, this definition is too simplistic.
 
-Formally, a Client is a set of cryptographic objects composed by
+Formally, a client is a set of cryptographic objects composed by
 public values such as a name (an identity), a public encryption key
-and a public signature key. Ownership of a Client by a user is
+and a public signature key. Ownership of a client by a user is
 determined by the fact that the user has knowledge of the
-associated secret values. When a Client is part of a Group, it is
+associated secret values. When a client is part of a Group, it is
```

Comment:
Choice about capitalization, usually a generality about clients should not be capitalized while an « action » of a specific client should.